R/render.R

```diff
@@ -49,3 +51,12 @@ render.list <- function(data, filename, ..., template){
   }
 
 }
+
+add_content <- function(x, old){
```

Comment:
this allows meddle to evaluate a function from the yaml and handle assigning the response a name. For example, the `metadata-date` below